client/src/components/v3/dashboard/V3SpeedDistanceChart.tsx

```diff
@@ -0,0 +1,38 @@
+import { Sensor, useSensorData } from 'api/common/data';
+import SpeedDistanceChart from 'components/common/charts/SpeedDistanceChart';
+import React, { useEffect, useState } from 'react';
+import { ChartPoint } from 'types/chart';
+import { ReedDistanceRT, ReedVelocityRT } from 'types/data';
+
+
+
+/**
+ * Passes V3 data to the speed distance chart component
+ *
+ * @returns Component
+ */
+export function V3SpeedDistanceChart() {
+    // Used to store the data points
+    const [data, setData] = useState<ChartPoint[]>([]);
+
+    // Speed
+    const point = useSensorData(3, Sensor.ReedVelocity, ReedVelocityRT);
+    const distance = useSensorData(3, Sensor.ReedDistance, ReedDistanceRT);
```

Comment:
Do we want to implement a multi sensor data hook? Currently, the usage is finding the correct sensor from the notion mqtt topics page.
Not sure if that's something we want in the MVP or not.